src/main/java/dev/vasyl/proj/controller/BookController.java

```diff
@@ -20,33 +23,45 @@
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/api/books")
+@Tag(name = "Book manager",
+        description = "API for managing books")
 public class BookController {
 
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books by page",
+            description = "Get all books by page params or/and sort params")
+    public List<BookDto> getAll(Pageable pageable) {
```

Comment:
It's better to return Page here
Also you can use 
```suggestion
    public List<BookDto> getAll(@ParameterObject @PageableDefault Pageable pageable) {
```